haphilipsjs/__init__.py

```diff
@@ -972,6 +974,20 @@ async def setScreenState(self, state):
                 return True
         return False
 
+    async def getHueLampState(self):
+        r = await self.getReq("HueLamp/power")
+        if r:
```

Comment:
This should check json feature

Reply:
I dont understand - I have reused code from other get's - as I can see getReq() will check if response is json, and if not it will return empty dict, so additional json checks are not necessary.. But I can add something like:
```python
if r and "power" in r:
```
It will be OK then?

Reply:
I don't want the request to be done at all if the feature list does not contain the field for hue. The TVs are crash prone, more requests crash them more often.

Reply:
Oh, ok, I unterstand now - TV's suppoerted features from jsonfeatures :) Added